### PR TITLE
App Banner: Reduce time out from a year to a month

### LINKED ISSUE #20626: Change App Banner dismissal time to less than a year
We're going to experiment with a shorter time for showing the app banner again (let's start with a month)

All we need to do is tweak this to be a month instead of a year in milliseconds.

https://github.com/Automattic/wp-
calypso/blob/9dac69a6c4f5fbb44dbb5b2ddcf9bb36559e3e3c/client/blocks/app-banner/utils.js#L67

### PR DESCRIPTION
**Pull request by kwonye**
Fixes #20626 

Seeing the banner once a month shouldn't be too annoying. This also gives the apps two releases in between each timeout.

The value is simply `ONE_WEEK_IN_MILLISECONDS * 4`

## Testing instructions

Unfortunately I'm not aware of how to change the clock on Chrome itself, so not sure how to verify the timeout is correct. No logic is changed however, so it should behave exactly as before.

### CODE REVIEW

File: client/blocks/app-banner/utils.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ export const READER = 'reader';
 export const STATS = 'stats';
 export const ALLOWED_SECTIONS = [ EDITOR, NOTES, READER, STATS ];
 export const ONE_WEEK_IN_MILLISECONDS = 604800000;
-export const ONE_YEAR_IN_MILLISECONDS = 31540000000;
+export const ONE_MONTH_IN_MILLISECONDS = 2419200000;
```
Comment (gwwar) [inline client/blocks/app-banner/utils.js]: Maybe leave a comment of `//28 days`?

**Comment by kwonye** [comment]
Thanks @gwwar, done!

**Comment by gwwar** [comment]
> Unfortunately I'm not aware of how to change the clock on Chrome itself

JS uses your local system time, so you can time travel by updating the mac setting. Don't forget to change it back though :)

**Comment by kwonye** [comment]
Thanks @gwwar!

> Don't forget to change it back though :)

99% chance I would've been stuck a month behind for at least two days 😛

**Pull request merged**